README.md

```diff
@@ -157,8 +159,8 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
   - [Testing](#testing)
     - [Testing Frameworks](#testing-frameworks)
     - [Mock](#mock)
-    - [Fuzzing and delta-debugging/reducing/shrinking](#fuzzing-and-delta-debuggingreducingshrinking)
-    - [Selenium and browser control tools](#selenium-and-browser-control-tools)
+  - [Fuzzing and delta-debugging/reducing/shrinking](#fuzzing-and-delta-debuggingreducingshrinking)
+  - [Selenium and browser control tools](#selenium-and-browser-control-tools)
```

Comment:
These two categories belong to **`Testing`**.

Reply:
@yassinebenaid that's because it's `## Fuzzing and delta-debugging/reducing/shrinking`  but it should be `### Fuzzing and delta-debugging/reducing/shrinking`. I will fix that!